pandas/tests/indexes/test_category.py

```diff
@@ -269,28 +269,19 @@ def f(x):
                                   ordered=False)
         tm.assert_index_equal(result, exp)
 
-    def test_where(self):
+    @pytest.mark.parametrize('klass', [list, tuple, np.array, pd.Series])
+    def test_where(self, klass):
```

Comment:
we should prob move all of the test_where test to test_base and use the indices fixture to avoid the code repetition (new issue though)